lab-miketreat/test/greeting-test.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const greeting = require('../lib/greeting.js');
+const assert = require('assert');
+
+
+// For passing args during testing, use npm run test test <name> where <name> is the arguement you want to pass into the greet function.
+
+// This test will check if the value of process.argv is undefined and fail if it is.
+describe('greeting.js', function(){
+  describe('#greet', function(){
+    it('Success = defined, Fails = undefined. process.argv[3]. This succeeds if the value is defined and it fails if it is undefined.', function(){
```

Comment:
Generally we don't test process.argv or cli. Normally you would mock data, to test the actual functionality, like are you getting the right name back, are returning a string, things like that. As of right now, I would change your test to mocked data so that they can pass.